include/seqan/store/store_contig.h

```diff
@@ -192,13 +192,13 @@ struct ContigFile
     static const TId INVALID_ID;
 
     CharString        fileName;
-    AutoSeqFormat    format;
+    //AutoSeqFormat    format; // is deprecated
```

Comment:
Well I have no idea where this is used, since commenting it out did not lead to any warnings or errors, so I didn't know if I could delete it?